src/main/java/entities/SessionStorage.java

```diff
@@ -0,0 +1,38 @@
+package entities;
+
+import java.util.ArrayList;
+
+public class SessionStorage {
+
+    private final ArrayList<MonthlyStorage> monthlyData;
+    private final ArrayList<Expense> recurData;
+
+    /**
+     * Creates a new SessionStorage and assigns empty values to both its instance attributes
+     */
+    public SessionStorage(){
+        this.monthlyData = new ArrayList<>();
+        this.recurData = new ArrayList<>();
+    }
+
+    public void setMonthlyData(MonthlyStorage month){
```

Comment:
can we change the name of this one to addMonthlyData? Cause this one is adding something to the array instead of setting it